### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by johnnagro** [AtomLinter/linter-rubocop#39]
Atom editor produced this warning message:

Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:1060:19)
LinterRubocop.destroy (/Users/jnagro/.atom/packages/linter-rubocop/lib/linter-rubocop.coffee:36:16)
LinterView.remove (/Users/jnagro/.atom/packages/linter/lib/linter-view.coffee:252:6)
<unknown> (/Users/jnagro/.atom/packages/linter/lib/linter-view.coffee:124:7)
```
